core-web/libs/dotcms-models/src/lib/dot-workflow.model.ts

```diff
@@ -39,3 +39,38 @@ export interface DotProcessedWorkflowPayload {
     pathToMove: string;
     contentlet: Record<string, unknown>;
 }
+
+export interface DotCMSWorkflowStatus {
+    scheme: DotCMSWorkflow;
+    step: WorkflowStep;
+    task: WorkflowTask;
+}
+
+export interface WorkflowStep {
+    creationDate: number;
+    enableEscalation: boolean;
+    escalationAction: string | null;
+    escalationTime: number;
+    id: string;
+    myOrder: number;
+    name: string;
+    resolved: boolean;
+    schemeId: string;
+}
+
+export interface WorkflowTask {
+    assignedTo: string;
+    belongsTo: string | null;
+    createdBy: string;
+    creationDate: number;
+    description: string;
+    dueDate: string | null;
+    id: string;
+    inode: string;
+    languageId: number;
+    modDate: number;
+    new: boolean;
```

Comment:
can we use `new`, is not a JavaScript reserved word?

Reply:
Is a reserved word, and ChatGPT says that is not recommended to use (As a property, probably we dont have errors, but isnt recommende)
And i think this is a Response for a endpoint, so we cannot change this word